eslint.config.mjs

```diff
@@ -0,0 +1,1059 @@
+/* eslint-disable @stylistic/js/max-len */
+
+import { createRequire } from 'node:module';
+import { fileURLToPath, URL } from 'node:url';
```

Comment:
URL is a global variable

```suggestion
import { fileURLToPath } from 'node:url';
```

Reply:
Yes, but by default ESLint doesn't know any globals. I didn't want to configure it specifically for this file.